cpp/include/cudf/lists/drop_list_duplicates.hpp

```diff
@@ -27,38 +29,65 @@ namespace lists {
  * @file
  */
 
+/*
+ * @brief Flag to specify which entry to keep when removing the duplicate entries from a repeated
+ * sequence.
+ */
+enum class keep_policy {
```

Comment:
Can we reuse/extend https://github.com/rapidsai/cudf/blob/e63f45543924bfef79f5fa5f7d3bd88c574b3a7f/cpp/include/cudf/stream_compaction.hpp#L210 ?

Reply:
Yes we can. I will do it.